package.json

```diff
@@ -39,6 +40,7 @@
         "react-swipeable-views-utils": "^0.14.0",
         "sass-mediaqueries": "^1.6.1",
         "save-as": "^0.1.8",
+        "screen-size-detector": "^1.0.4",
```

Comment:
if this was a new NPM package that was installed please uninstall it.  If you need any help with this please let me know.